src/components/common/DynamicTabs/DynamicTabs.tsx

```diff
@@ -18,11 +18,11 @@ import React, { Fragment, useEffect, useRef, useState } from 'react'
 import { useHistory } from 'react-router-dom'
 import Tippy from '@tippyjs/react'
 import { Dayjs } from 'dayjs'
-import { stopPropagation, ConditionalWrap, noop, OptionType } from '@devtron-labs/devtron-fe-common-lib'
+import { stopPropagation, ConditionalWrap, noop, OptionType, DynamicTabType } from '@devtron-labs/devtron-fe-common-lib'
```

Comment:
Changes context: had to add a hideName prop in DynamicTabs which would hide the name from tab title